print(1)
try:
	i = 10/0
except:
	print('some thing went wrong')
print(2)